caikit/core/data_model/training_info.py

```diff
@@ -0,0 +1,35 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Common data model enum used for reporting training information
+"""
+
+# Standard
+from typing import List
+
+# Local
+from ..toolkit.wip_decorator import Action, WipCategory, work_in_progress
+from .dataobject import DataObjectBase, dataobject
+from .package import PACKAGE_COMMON
+from .training_status import TrainingStatus
+
+
+@work_in_progress(action=Action.WARNING, category=WipCategory.BETA)
```

Comment:
I'm pretty tempted to yank all of these decorators off since technically the whole project is still in `beta` and it's pretty noisy/confusing to have them just on train related stuff

Reply:
True. I'll get that removed

Reply:
I think I added these explicitly for the initial drop of this stuff where we only had local training and none of it supported any sort of distribution. Now that we have real distributed trainers, I think it's reasonably "no longer work in progress"